_summaries/bendale2015towards_1.md

```diff
@@ -1,19 +1,42 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: bendale2015towards 
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: ywen666 
+score: 9
 ---
 
-TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+## Motivation
 
-## TL;DR
-* Three
-* Bullets
-* To highlight the core concepts
+**Problem** It is easy to generate images that humans would never classify as a particular object class, yet networks classify such images high confidence as that given class. The closed set nature of deep networks forces them to choose from one of the known classes leading to such artifacts. However, recognition in the real world is open set, i.e. the recognition system should reject unknown/unseen classes at test time.
+
+The SoftMax layer is a significant component of the problem because of its closed nature. The authors propose an alternative, OpenMax, which extends SoftMax layer by enabling it to predict an unknown class.
+
+## OpenMax
+<img width="400px" src="bendale2015towards_1a.png"/>  
+
+**Design**: “open space risk” should be measured in feature space, rather than in pixel space. The key question: “is there a feature space, ideally a layer in the deep network, where these adversarial images are far away from training examples.
+
+<img width="800px" src="bendale2015towards_1b.png"/>  
+
+* Multi-class Meta-Recognition
+  * Each class is represented as a point, a mean activation vector (MAV) with the mean computed over only the correctly classified training examples
+  * Given the MAV and an input image, the authors measure distance between them. They directly threshold distance to determine an overall maximum distance threshold.
+  * In particular, on line 3 of Alg. 1 we use the libMR FitHigh function to do Weibull fitting on the largest of the distances between all correct positive training instances and the associated $µ_i$. This results in a parameter $\rho_i$ , which is used to estimate the probability of an input being an outlier with respect to class i.
```

Comment:
1. Here as well as elsewhere, you should enclose the math symbols inside double dollar signs ($$....$$)
2. Probably use j for indexing to be consistent with the algorithm?

Reply:
done